src/actions/get-oracles-transfer-events.ts

```diff
@@ -0,0 +1,49 @@
+import db from "src/db";
+import logger from "src/utils/logger-handler";
+import {Network_v2} from "@taikai/dappkit";
+import {EventsProcessed,EventsQuery,} from "src/interfaces/block-chain-service";
+import {OraclesChangedEvent} from "@taikai/dappkit/dist/src/interfaces/events/network-v2-events";
+import {EventService} from "../services/event-service";
+import {BlockProcessor} from "../interfaces/block-processor";
+import BigNumber from "bignumber.js";
+import { handleCurators } from "src/modules/handle-curators";
+
+export const name = "getOraclesTransferEvents";
+export const schedule = "*/30 * * * *";
+export const description = "Sync transfer oracles data and update council's count";
+export const author = "marcusviniciusLsantos";
+
+
+async function handleTransfers(addresses: string[], councilAmount: string, networkId: number, service) {
+ return Promise.all(addresses.map(async address => {
```

Comment:
you don't need to await on a `Promise.all` since its already awaiting a promise, 

```js
Promise.all(
  addresses.map(address =>
    service.Actor.getOraclesOf(address)
      .then(votes => handleCurators(address, votes, councilAmount, networkId)
    )
  )
)
```